### PR TITLE
chore(type-definitions): until

### PR DESCRIPTION
**Pull request by nissoh**

### CODE REVIEW

**Comment by briancavalier** [comment]
Thanks @nissoh! I notice that skipUntil should also be named since. Would you mind including that change in this PR, too?

**Comment by nissoh** [comment]
also noticed `until` in curried form will only accept `endSignal` type. @TylorS ?

File: packages/core/type-definitions/combinator/timeslice.d.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import { Stream } from '@most/types';
 
 export function until<A>(signal: Stream<any>, s: Stream<A>): Stream<A>;
-export function until<A>(signal: Stream<A>): (s: Stream<A>) => Stream<A>;
+export function until<A>(signal: Stream<any>): (s: Stream<A>) => Stream<A>;
```
Comment (nissoh) [inline packages/core/type-definitions/combinator/timeslice.d.ts]: also noticed until in curried form will only accept endSignal type. @TylorS ?

**Comment by briancavalier** [comment]
Yep, `any` seems correct there.  Good catch.  This LGTM.  Any concerns, @TylorS?

File: packages/core/type-definitions/combinator/timeslice.d.ts (hunk 1 above)
Comment (TylorS) [inline packages/core/type-definitions/combinator/timeslice.d.ts]: It takes a `signal` then returns a function that takes a `stream` returning a new stream that will end when the signal emits.

**Pull request merged**